tests/ui/llm_studio.feature

```diff
@@ -0,0 +1,30 @@
+Feature: LLM Studio
+
+    Background: LLM Studio user
+        Given LLM Studio home page is opened
+        When I login to LLM Studio
+        Then I see the home page
+
+    Scenario: Import dataset using filesystem
+        When I upload dataset with path oasst/train_full.pq and name test-dataset
+        Then I should see the dataset test-dataset
+
+    Scenario: Delete dataset
+        When I delete dataset test-dataset
```

Comment:
scenario starts assuming there's a dataset already, i.e. depends on previous scenario.
in general, that's not a good idea. if you need a flow, join to scenarios.
if you insist on having two scenarios, make them independent on the outcome of each other.

Reply:
Oh intresting, lets combine them in that case @mnezh